packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/quanticRefineModalContent.js

```diff
@@ -0,0 +1,219 @@
+} from 'c/quanticHeadlessLoader';
+import filters from '@salesforce/label/c.quantic_Filters';
+import clearAllFilters from '@salesforce/label/c.quantic_ClearAllFilters';
+
+/** @typedef {import("coveo").SearchStatus} SearchStatus */
+/** @typedef {import("coveo").SearchEngine} SearchEngine */
+/** @typedef {import("coveo").BreadcrumbManager} BreadcrumbManager */
+
+const COMMON_FACET_PROPERTIES = ['facetId', 'field', 'label'];
+const DEFAULT_FACET_PROPERTIES = [
+  'numberOfValues',
+  'sortCriteria',
+  'noSearch',
+  'displayValuesAs',
+  'noFilterFacetCount',
+  'injectionDepth',
+];
+const NUMERIC_FACET_PROPERTIES = [
+  'numberOfValues',
+  'sortCriteria',
+  'rangeAlgorithm',
+  'withInput',
+  'formattingFunction',
+];
+const CATEGORY_FACET_PROPERTIES = [
+  'basePath',
+  'noFilterByBasePath',
+  'noFilterFacetCount',
+  'delimitingCharacter',
+  'numberOfValues',
+  'sortCriteria',
+  'withSearch',
+];
+const TIMEFRAME_FACET_PROPERTIES = [
+  'withDatePicker',
+  'noFilterFacetCount',
+  'injectionDepth',
+];
+
+/**
+ * The `QuanticRefineModalContent` component displays a copy of the search interface facets and sort components. This component is intended to be displayed inside a modal to assure the responsiveness when the search interface is displayed on smaller screens.
+ *
+ * @category Search
+ * @example
+ * <c-quantic-refine-modal-content engine-id={engineId} hide-sort></c-quantic-refine-modal-content>
+ */
+export default class QuanticRefineModalContent extends LightningElement {
+  labels = {
+    filters,
+    clearAllFilters,
+  };
+
+  /**
+   * The ID of the engine instance the component registers to.
+   * @api
+   * @type {string}
+   */
+  @api engineId;
+  /**
+   * Whether the Quantic Sort component should be hidden.
+   * @api
+   * @type {boolean}
+   */
+  @api hideSort;
+
+  /**@type {object} */
+  data;
+
+  connectedCallback() {
+    registerComponentForInit(this, this.engineId);
+  }
+
+  renderedCallback() {
+    initializeWithHeadless(this, this.engineId, this.initialize);
+  }
+
+  disconnectedCallback() {
+    this.unsubscribeSearchStatus?.();
+  }
+
+  /**
+   * @param {SearchEngine} engine
+   */
+  initialize = (engine) => {
+    this.searchStatus = CoveoHeadless.buildSearchStatus(engine);
+    this.breadcrumbManager = CoveoHeadless.buildBreadcrumbManager(engine);
+    this.unsubscribeSearchStatus = this.searchStatus.subscribe(() =>
+      this.gatherFacets()
+    );
+  };
+
+  /**
+   * Gather all facets registred from the Quantic store.
+   * @returns {void}
+   */
+  gatherFacets() {
+    if (!this.hasFacets) {
+      this.data = getAllFacetsFromStore(this.engineId);
+    }
+  }
+
+  /**
+   * Deselects all the filters.
+   * @returns {void}
+   */
+  clearAllFilters() {
+    this.breadcrumbManager.deselectAll();
+  }
+
+  /**
+   * Returns the data needed to create a copy of the numeric facet.
+   * @param {HTMLElement} facetElement
+   * @returns {object}
+   */
+  static toNumericFacet(facetElement) {
```

Comment:
I think these functions don't have to be `static`. It is best to keep them as private instance functions at this moment.